dashboard/lib/standards_autocomplete.rb

```diff
@@ -0,0 +1,18 @@
+class StandardsAutocomplete < AutocompleteHelper
+  def self.get_search_matches(query, limit, framework_id = nil)
+    limit = format_limit(limit)
+    rows = Standard.limit(limit)
+    rows = rows.where(framework_id: framework_id) if framework_id
+
+    shortcode_rows = rows.where('shortcode like ?', "#{query}%").all
+    return shortcode_rows.map(&:summarize_for_lesson_edit) unless shortcode_rows.empty?
+
+    return [] if query.length < MIN_WORD_LENGTH
```

Comment:
Elijah suggested moving this to the top of the method for programming expression so you don't do other queries if you don't need to.

Reply:
I think the difference here is that if there is a query like "6." as appears in the test cases, then I do want to return results for shortcodes starting with that prefix, whereas I wouldn't want to do it for fulltext because fulltext index only works on words that are 3+ characters.

Reply:
Oh that makes sense!